src/Helper.php

```diff
@@ -114,10 +114,25 @@ public static function get_shipment_carriers() {
      * @return array List of aliases.
      */
     public static function get_aliases() {
-        return [
-            'delivered' => __( 'Delivered', 'trackmage' ),
-            'shipped'   => __( 'Shipped', 'trackamge' ),
-        ];
+        $workspaceId = get_option( 'trackmage_workspace', 0 );
+        $aliases = [];
+
+        if($workspaceId !== 0){
+            try {
+                $client = Plugin::get_client();
+                $guzzleClient = $client->getGuzzleClient();
+                $response = $guzzleClient->get("/workspaces/{$workspaceId}/order_statuses");
+                $content = $response->getBody()->getContents();
+                $data = json_decode($content, true);
+            } catch( ApiException $e ) {
+                // Do nothing. We will return an empty array.
+            }
+            foreach ($data['hydra:member'] as $status) {
```

Comment:
in case of exception $data will be undefined. maybe add `return []` inside the `catch` part?

Reply:
in that case I think we can move foreach into try.

Reply:
Ok